src/styles/index.scss

```diff
@@ -1,3 +1,17 @@
+@import './utils/variables';
+@import './blocks/card';
+@import './blocks/header';
+@import './blocks/main';
+
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+a,
+p {
+  margin: 0;
+  padding: 0;
 }
```

Comment:
You have reset the margin and padding for 'a' and 'p' tags. This might not be the best approach because it can lead to inconsistencies in styles. It's better to reset these properties for all elements at once using a universal selector (*), or even better, use a CSS reset or normalize library. This way, you'll have a clean slate to start with and you won't have to worry about browser default styles interfering with your design.